core/src/components/menu/menu.tsx

```diff
@@ -619,12 +620,12 @@ export class Menu implements ComponentInterface, MenuI {
       // emit open event
       this.ionDidOpen.emit();
 
-      // focus menu content for screen readers
-      if (this.menuInnerEl) {
-        this.focusFirstDescendant();
-      }
-
-      // setup focus trapping
+      /**
+       * Set up focus trapping. Focus the host element to start with
+       * instead of the first descendant to avoid the scroll position
+       * jumping to that element right away.
+       */
+      this.el.focus();
```

Comment:
Should this check to see if the focused element is already inside of `ion-menu`? If it is, we probably do not want to change the focus.

Reply:
This is only when opening the menu, so the focus should be on whatever button etc. triggered that (which won't be inside it).

Reply:
What happens if a user manually shifts focus? (I.e. do we still respect that focus?)

Reply:
Shifts focus to where? If you hit tab after opening the menu, it'll move to the contents and trap your focus, but not sure if that's what you meant.

Reply:
If a user manually focuses an element inside of `ion-menu`, I would expect that the focus is preserved when opening the menu. For example: Someone clicks the menu button and focuses a button inside the menu as it is opening.